test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -462,6 +467,12 @@ def test_avs_failed_authorize
     assert_equal 'request_invalid: card_number_invalid', response.message
   end
 
+  def test_invalid_shipping_address
+    response = @gateway.authorize(@amount, @credit_card, shipping_address: address.update(country: 'Canada'))
+    assert_failure response
+    assert_equal 'request_invalid: address_country_invalid', response.message
```

Comment:
The gateway responded with this error message when I ran the remote test 🤔 
```suggestion
    assert_equal 'request_invalid: country_address_invalid', response.message
```